examples/getting-started-app/README.md

```diff
@@ -17,17 +17,22 @@ Features:
 
 ## Setting up the environment
 
+Before you start developing with cartridge, you need to install several utilities:
```

Comment:
Please highlight the cartridge with `` = `cartridge` (for consistency with Russian version)